website/src/main/java/com/google/step/servlets/DataServlet.java

```diff
@@ -29,4 +34,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreServiceFactory.getDatastoreService().put(getOrgEntityFrom(request));
+    response.sendRedirect("/index.html");
+  }
+
+  private Entity getOrgEntityFrom(HttpServletRequest request) {
+    Entity newOrganization = new Entity("Organization");
+    newOrganization.setProperty("name", request.getParameter("orgName"));
+    newOrganization.setProperty("webLink", request.getParameter("webLink"));
+    newOrganization.setProperty("donateLink", request.getParameter("donateLink"));
+    newOrganization.setProperty("about", request.getParameter("about"));
```

Comment:
Optional: add error checking for if any or all of the entries that a user entered is empty